src/scipp/plotting/tools.py

```diff
@@ -85,7 +85,7 @@ def parse_params(params=None, defaults=None, globs=None, array=None):
         parsed["cmap"] = LinearSegmentedColormap.from_list(
             "tmp", [parsed["color"], parsed["color"]])
     else:
-        parsed["cmap"] = copy(cm.get_cmap(parsed["cmap"]))
+        parsed["cmap"] = copy(matplotlib.colormaps[parsed["cmap"]])
```

Comment:
Note that this is not supported in mpl < 3.4 (I think). So either we do this and put a lower bound on the mpl requirement, or we filter out the warning for now.

Reply:
We would have to filter the warning in downstream code as well, see scippneutron. Or we filter it just in this call until it becomes an actual deprecation warning. But then we have to revisit this. Is there a problem with a lower bound on mpl?

Reply:
I think lower bound is fine